arr-of-first-element.js

```diff
@@ -1,13 +1,8 @@
-/** 
- * @param {object} a - array
- * @return {object} firstIndex - returns the first element of an array
- */
-
- function arrOfFirst(a) {
-    var firstIndex = a[0]; 
+ function arrOfFirst(arr) {

```

Comment:
/** 
 * Returns the first element in an array
 * @param {object} arr - mixed array
 * @returns {object} firstIndex - returns the first element of mixed array
 */